gems/ruby-openid/CVE-2019-11027.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
I had purposely not included this one, as https://github.com/openid/ruby-openid/issues/122 disputes it. Let's leave this out.

Reply:
git rm'd from PR.